red.py

```diff
@@ -52,7 +52,8 @@ def __init__(self, *args, **kwargs):
         self.uptime = datetime.datetime.now()
         self._message_modifiers = []
         self.settings = Settings()
-        super().__init__(*args, **kwargs)
+        prefix_man = lambda bot, msg: bot.settings.get_prefixes(msg.server)
```

Comment:
no docs?
although we don't really write docs on anything other than commands *shrug